notebooks/text-classification_multilabel.ipynb

```diff
@@ -0,0 +1,548 @@
+{
```

Comment:
<p>It'd be great to add hyperparameter search for <code>num_iterations</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/setfit/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
Yeah I agree this is out of scope (but a nice idea nonetheless!). Once #8 is implemented, we could revisit this :)